app/services/to_cocina/work/mapper.rb

```diff
@@ -57,9 +57,13 @@ def object_type
       def document?
         return false if content.content_files.empty?
 
-        content.content_files.all? do |content_file|
-          !content_file.hidden? && content_file.mime_type == 'application/pdf'
+        # All files are PDF or hidden
+        return false unless content.content_files.all? do |content_file|
+          content_file.hidden? || content_file.pdf?
         end
+
+        # There is at least one non-hidden PDF
+        content.content_files.any? { |content_file| !content_file.hidden? && content_file.pdf? }
```

Comment:
@justinlittman Any reason to be wary of this simplification? https://github.com/sul-dlss/hungry-hungry-hippo/pull/783/files